cli.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
```

Comment:
recommend to move cli.js into scripts folder in the root. it is kinda a typical pattern to store scripts in npm packages. but it is not super important right now.